packages/clerk-js/src/ui/primitives/gapPropertyCompat.ts

```diff
@@ -0,0 +1,13 @@
+export const createFlexGapPropertyIosCompat = (val: string) => {
+  // Resolve gap property compatibility issue on iOS 13
+  // This is not needed for css grid, only for flex
+  // TODO: Test whether we need to make the check more generic
+  if (navigator?.userAgent?.match(/(iphone|ipad).+os.+13.+safari/i)) {
+    return {
+      '& > * + *': {
+        marginTop: val,
+      },
```

Comment:
❓ We use the `Flex` primitive with `flex-direction: row`, will this `marginTop` work then?

Reply:
Very valid consideration, let me check how many uses we have for flex-direction

Reply:
Actually, the default direction is `row`